media/server/gstplayer/source/GstSrc.cpp

```diff
@@ -375,8 +366,26 @@ void GstSrc::setupAndAddAppArc(GstElement *element, GstElement *appsrc, GstAppSr
     m_gstWrapper->gstBinAdd(GST_BIN(element), appsrc);
 
     GstElement *src_elem = appsrc;
+
+    // Configure and add decryptor
+    GstElement *decryptor = m_decryptorFactory->createDecryptorElement(nullptr, decryptionService);
+    if (decryptor)
+    {
+        GST_DEBUG("Injecting decryptor element %" GST_PTR_FORMAT, decryptor);
```

Comment:
Can you change loggers in this function from _GST_XXX(...)_ to _GST_XXX_OBJECT(element, ...)_ ?

Reply:
Done.